src/python/grpcio_tests/tests_py3_only/unit/_simple_stubs_test.py

```diff
@@ -19,19 +19,21 @@
 
 _MAXIMUM_CHANNELS = 10
 
-os.environ["GRPC_PYTHON_MANAGED_CHANNEL_EVICTION_SECONDS"] = "1"
+os.environ["GRPC_PYTHON_MANAGED_CHANNEL_EVICTION_SECONDS"] = "2"
```

Comment:
Is there a reason to increase the eviction time?

Reply:
The `wait_for_ready` test consistently fails when this is set to 1 second due to the channel being evicted before it comes ready. At 2 seconds, I see 0 failures out of 1000. I believe there is a hardcoded constant of 1 second to go from connecting to transient failure in core, but I haven't dug to figure out where it is.

Reply:
Just a thought. Should we start counting eviction seconds only if the channel is in `READY` state? It can be done by the `_create_channel` function also calls `channel_ready_future`.

Reply:
Interesting idea. But then a channel that never comes ready would never be evicted., right? That seems like a recipe for unbounded memory usage. The default eviction time is *much* higher than 1 second. I expect that this failure mode will only be observed when users are attempting to reach a backend that is actually down/misbehaving.

You could weaken the idea in order to solve that problem. For example, you could set the timer back to 0 when the channel comes ready. But you couldn't do this *every* time the channel comes ready. In this case, a flapping channel would also never be evicted. So it would be reasonable to reset the timer to 0 the *first* time the channel comes ready, but at that point, it doesn't seem to have much value. The amount of time connecting to the backend is likely to be in the noise compared to the amount of time spent servicing the request and returning the response.

Thoughts?

Reply:
Yeah, it might cause a channel never be evicted, I missed that. Either way seems not yielding a clear benefit. Then, it might not be worthwhile to change the timeout algorithm.